internal/pkg/manifest/validate.go

```diff
@@ -964,13 +970,13 @@ func (qs QueueScaling) Validate() error {
 	if qs.IsEmpty() {
 		return nil
 	}
-	if qs.AcceptableLatency == nil {
+	if qs.AcceptableLatency == nil && qs.AvgProcessingTime != nil {
 		return &errFieldMustBeSpecified{
 			missingField:      "acceptable_latency",
 			conditionalFields: []string{"msg_processing_time"},
 		}
 	}
-	if qs.AvgProcessingTime == nil {
+	if qs.AvgProcessingTime == nil && qs.AcceptableLatency != nil {
```

Comment:
i think this case is the same as the above one?

Reply:
Since `IsEmpty()` for queue scaling now includes checking the cooldown, I have to specify that the `AcceptableLatency` isn't nil (and vise versa) as it should be valid for only the cooldown to be defined (it just won't do anything).

Reply:
ah, gotcha - i was missing the `!=` nil switch :)